### DISCUSSION TITLE
[firebase_database] FirebaseAnimatedList should have a 'emptyListChild'

### DISCUSSION THREAD

**Discussion by kroikie** [firebase/flutterfire#5568] [Feature request]
## Current behavior

When using the firebaseAnimatedList for user generated content, the first time the user loads the application and there's no content, is just not optimal to load an empty container.

## Expected behavior

The ability to show a custom widget when the list is empty

**Comment by kroikie** [comment]
@Panthro

The issue at https://github.com/flutter/flutter/issues/15480 has been closed and moved here. Future collaboration on this issue will be done here.

**Reply by MrOwaisYosuf** [reply]
There must be an option for "ifEmpty" inside FirebaseAnimatedList, so that empty state can also be handled easily.

**Comment by devroldy** [comment]
Any progress on this? There needs to be a custom widget for when FirebaseAnimatedList has no value in the list
